.gitignore

```diff
@@ -125,6 +125,3 @@ dmypy.json
 
 # Pycharm project settings
 .idea/
-
```

Comment:
We want to track this `.yaml` file now, since we're not relying on a `.in` file.